crates/turborepo-lib/src/package_graph/mod.rs

```diff
@@ -53,6 +53,13 @@ impl WorkspaceInfo {
             .expect("at least one segment")
     }
 
+    pub fn task_log_path(&self, task_id: &TaskId) -> AnchoredSystemPathBuf {
```

Comment:
Moving the logic out of the task cache so we can reuse it when constructing the `TaskSummary` without plumbing it out of the task cache.